deploy/kubernetes/4-collector-config.yaml

```diff
@@ -303,47 +307,41 @@ data:
       ##########################################################################
 
       # auto-discover kube DNS
-      - name: kube-dns-discovery
-        type: prometheus
-        selectors:
-          images:
-          - '*kube-dns/sidecar*'
-          labels:
-            k8s-app:
-            - kube-dns
-        port: 10054
-        path: /metrics
-        scheme: http
-        prefix: kube.dns.
-        filters:
-          metricAllowList:
-          - 'kube.dns.http.request.duration.microseconds'
-          - 'kube.dns.http.request.size.bytes'
-          - 'kube.dns.http.requests.total.counter'
-          - 'kube.dns.http.response.size.bytes'
-          - 'kube.dns.kubedns.dnsmasq.*'
-          - 'kube.dns.process.*'
```

Comment:
Would like clarification for the sake of the newly opened PR #366: 
What is the reason for commenting the dns and coredns metrics here? Need to know if the same needs to be done in the new PR #366 as well.